api/views.go

```diff
@@ -88,7 +89,7 @@ type ViewConnectionInput struct {
 	Filter         graphql.String `json:"filter"`
 }
 
-func (c *Views) Create(name, description string, connections map[string]string) error {
+func (c *Views) Create(name, description string, connections map[string][]string) error {
```

Comment:
Looks like you're also updating the views creation method, causing compilation to fail since `cmd/humioctl/views_create.go` hasn't been updated to use the new type:
```
$ make clean all
go clean
go build -o bin/humioctl ./cmd/humioctl
# github.com/humio/cli/cmd/humioctl
cmd/humioctl/views_create.go:50:56: cannot use connections (variable of type map[string]string) as map[string][]string value in argument to client.Views().Create
make: *** [bin/humioctl] Error 1

```

Reply:
Oof, I am sorry for that. I thought I had compiled and done some manual testing after the latest modifications.